requirements/ci.txt

```diff
@@ -1,6 +1,6 @@
 -r towncrier.txt
 build==1.0.3
-coverage==7.3.3
+coverage==7.6.1
```

Comment:
Would you please bump this in a separate PR

Reply:
Done: https://github.com/aio-libs/frozenlist/pull/597